api/osf_api.py

```diff
@@ -1,7 +1,7 @@
-import datetime
 import json
 import logging
 import os
+from datetime import datetime
```

Comment:
The `datetime` import fix in this file will conflict with Ramya's PR, depending which PR gets merged first.